data/datum.go

```diff
@@ -21,6 +21,7 @@ type Datum interface {
 	GetPayload() *metadata.Metadata
 
 	GetType() string
+	GetActive() bool
```

Comment:
When the function returns a boolean it's better to prefix the attribute with `Is` instead of `Get` so it reads naturally when you use it in an if statement - e.g. `if datum.IsActive() { ... }`

Reply:
Debated this while I was writing it, as SetActive's opposite should be GetActive, even if it makes very little wording sense.
But happy to change it with a vote to weigh it down the other way, done.